### DISCUSSION TITLE
How is this lesson different from other blockchain lessons? e.g. Udacity/ Coursera blockchain course

### DISCUSSION THREAD

**Discussion by sojiu** [smartcontractkit/full-blockchain-solidity-course-py#1530] [Q&A]
How is the lesson different from Udacity course and other courses on other websites? Thanks! 
https://www.udacity.com/course/blockchain-developer-nanodegree

**Comment by javierdash** [comment]
Hello, the link you send does not work. Anyway, they are different courses. Different teachers and surely different ways of explaining it. This course is very complete and on top of that the community is great and they can help you with whatever you need :)

**Comment by cromewar** [comment]
As far as I seen, this course has more advanced material than other course which generally just cover the basics.

**Comment by fantauzzi** [comment]
Udacity's is in Javascript, also the content is, in my view, obsolete: versions of the tools and libraries that are way too old. I found the projects and the syllabus very interesting. But the supply chain project skeleton code is half baked, and found the code walkthroughs to be a snoozefest. 

With this course you must find your way to practice, which notoriously is the only way to understand and retain the content. What works for me is to take bullet points of what Master Patrick does, for half a section at a time, and then try to redo what he did without looking at the video and the Github again (as much as I can). Then I walk through my source code with a debugger, also try to implement some changes (e.g. deploying to Goerli instead of Rinkeby, using Alchemy instead of Infura, etc.).
